load.php

```diff
@@ -65,6 +65,7 @@ function get_config() {
 		'aws-ses-wp-mail' => true,
 		'tachyon'         => true,
 		'cavalcade'       => true,
+		'members'         => false,
```

Comment:
can we call this `private` maybe? to be more descriptive

Reply:
that just feels confusing to me from a developers perspective. But I'm happy to change it!

Reply:
perhaps we can get a better description, `members` is not meant to be "Members only plugin", but really "make site private", the fact the plugin is called "members only" doesn't really matter, we want it to be a label that makes sense when a developer wants to make a site private. However, maybe `logged-in-only` (which we can default to false) would also work?

Reply:
`members` isn't really WordPress user terminology, that's the main problem I think.

Reply:
Right, that makes sense to me. `logged-in-only` sounds fine for now.